build/gulpfile.hygiene.js

```diff
@@ -284,6 +288,23 @@ function hygiene(some) {
 
 		this.emit('data', file);
 	});
+
+	const localizeDoubleQuotes = es.through(function (file) {
+		const lines = file.__lines;
+		lines.forEach((line, i) => {
+			if (/localize/.test(line)) {
+				//console.error(file.relative + '(' + (i + 1) + ',1): Message parameter to localize calls should be double-quotes');
+				//errorCount++;
+			}
+			if (/localize\(['"].*['"],\s'.*'\)/.test(line)) {
```

Comment:
1. So, I think this'll miss if you have whitespace between the final ' and ), e.g. `localize('a', 'b' )`. Might cause us to miss a few cases?
2. Do we care about commented out lines?

Reply:
1. That'd flag the formatting hygiene task anyways so will end up being caught eventually
2. As in do we care about ignoring them? I don't really see a problem with including them - IMO we shouldn't be having commented lines with localize calls anyways so catching those isn't really a huge concern